nservicebus/azure-service-bus/performance-tuning.md

```diff
@@ -1,23 +1,11 @@
 ---
 title: Performance Tuning
+component: ASB
 tags:
  - Azure
  - Cloud
 reviewed: 2016-09-21
```

Comment:
should this reflect when I started, or when I ended :trollface: (just kidding, I'll update it)